akka-docs/src/test/scala/docs/persistence/state/PersistenceStatePluginDocSpec.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2009-2023 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.persistence.state
+
+import akka.Done
+import akka.actor.ActorSystem
+
+import akka.testkit.TestKit
+import com.typesafe.config._
+import org.scalatest.wordspec.AnyWordSpec
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+//#plugin-imports
+import akka.persistence._
+import akka.persistence.state.scaladsl.DurableStateUpdateStore
+import akka.persistence.state.scaladsl.GetObjectResult
+//#plugin-imports
+
+class PersistenceStatePluginDocSpec extends AnyWordSpec {
+  {
+    val providerConfig =
+      """
+        //#journal-plugin-config
+        # Path to the journal plugin to be used
+        akka.persistence.state.plugin = "my-journal"
+
+        # My custom journal plugin
+        my-journal {
+          # Class name of the plugin.
+          class = "docs.persistence.state.MyJournal"
+          # Dispatcher for the plugin actor.
+          plugin-dispatcher = "akka.actor.default-dispatcher"
+        }
+        //#journal-plugin-config
+      """
+
+    val system = ActorSystem(
+      "PersistenceStatePluginDocSpec",
+      ConfigFactory
+        .parseString(providerConfig)
+      //.withFallback(ConfigFactory.parseString(PersistencePluginDocSpec.config))
+    )
+    try {
+      Persistence(system)
```

Comment:
@patriknw Do you see a way to actually test if the custom plugin can be started, based on the config above? This call here seems to succeed even if I place some error like a wrong plugin class in the config above.

Reply:
```
val store = DurableStateStoreRegistry(system).durableStateStoreFor[DurableStateUpdateStore[Any]]("my-state-store")
```